HotSpot/SearchCell.h

```diff
@@ -0,0 +1,22 @@
+//
+//  SearchCell.h
+//  HotSpot
+//
+//  Created by aodemuyi on 7/16/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SearchCell : UITableViewCell
+
+@property (weak, nonatomic) IBOutlet UIImageView *searchTableImage;
```

Comment:
as discussed yesterday, IBOutlets should be in the implementation file, but the header file